text/exercise2/world_cup.md

```diff
@@ -1,12 +1,10 @@
 # Women's world cup
 
-The Women's World Cup stands as the pinnacle of global competition 
-in the NFL, showcasing the finest female athletes in intense matchups 
-on the gridiron. This prestigious tournament, organized by FIFA, 
-captivates audiences worldwide with its electrifying display of 
-athleticism, strategic plays, and nail-biting touchdowns. Teams from 
-across the globe, armed with skillful quarterbacks and powerful 
-offensive lines, battle for supremacy in this exhilarating event.
+The Women's World Cup is a spectacular showcase of athleticism, skill, 
+and passion, bringing together the finest female soccer players from 
```

Comment:
Move the sentence about soccer v football to this area, then be consistent and use the same term for the remainder of the article.